src/main/java/seedu/module/model/module/Deadline.java

```diff
@@ -0,0 +1,33 @@
+package seedu.module.model.module;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents a Person's deadline in the address book.
+ * Guarantees: immutable; is always valid
+ */
+public class Deadline {
+    public final String value;
```

Comment:
Try not to have public fields. [Reference](https://nus-cs2103-ay1920s1.github.io/website/coding-standards/java/intermediate.html#variables)